src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  value: [],
+};
+
+export const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    add: (state, action) => state.value.push(action.payload),
```

Comment:
[OPTIONAL]
- [ ] Nice work implementing the `add` action to add a book  to a state :clap: However React required  that state to be immutable. Therefore I suggested that instead of pushing to the `state.value` your code could look like this :point_down: 
```javascript
state.value = [
        ...state.value,
        action.payload
      ]
```